jp/test/main/resources/test.properties

```diff
@@ -0,0 +1,14 @@
+# comment
+test.foo.bar = true
+test.foo.baz = false
+
+test.bar = 42
+
+# another comment
+foo = string value
+
+test_state.default = West Midlands
+test_state.alternative = West Midlands
+
+test_town.default = Shrewsbury
+test_town.alternative = Shrewsbury
```

Comment:
I would recommend updating these so the unit tests use Tokyo.

Reply:
Updated. thx